src/NodaTime/Text/LocalTimePattern.cs

```diff
@@ -50,6 +50,32 @@ public sealed class LocalTimePattern : IPattern<LocalTime>
         /// <value>An invariant local time pattern which is ISO-8601 compatible, with no sub-second precision.</value>
         public static LocalTimePattern GeneralIso => Patterns.GeneralIsoPatternImpl;
 
+        /// <summary>
+        /// Gets an invariant local time pattern which is ISO-8601 compatible, with precision of just minutes.
+        /// This corresponds to the text pattern "HH':'mm':'ss".
+        /// </summary>
+        /// <value>An invariant local time pattern which is ISO-8601 compatible, with no sub-minute precision.</value>
+        public static LocalTimePattern HourMinuteIso => Patterns.HourMinuteIsoPatternImpl;
+
+        /// <summary>
+        /// Gets an invariant local time pattern which is ISO-8601 compatible, with a precision of just hours.
+        /// This corresponds to the text pattern "HH".
+        /// </summary>
+        /// <value>An invariant local time pattern which is ISO-8601 compatible, with no sub-hour precision.</value>
+        public static LocalTimePattern HourIso => Patterns.HourIsoPatternImpl;
+
+        /// <summary>
+        /// Gets an invariant local time pattern which can parse any ISO-8601 compatible value, (just hours;
```

Comment:
nit: patterns can all parse and format by definition, so it's a bit weird for this to say that it "can parse", when obviously it can format as well (but I agree that parsing is likely to be much more useful that formatting for this one).

Reply:
It's saying that it can parse *any* ISO-value, as opposed to the other patterns which only parse ones with a specific precision. The emphasis is intended to be on the latter part. Suggestions for anything that would make it clearer would be great :)

Reply:
That's fine, I think as-is is okay on reflection (and I can't come up with anything significantly better myself). As noted above, I think it's fine to put the emphasis on parsing for this pattern, since formatting with this is much less likely to be useful.